### ISSUE TITLE
rework refesher warning messages

### ISSUE THREAD

**Issue by xknat** [SkinsRestorer/SkinsRestorerX#724]
Issue is regarding following messages:
```
[13:03:43] [Server thread/INFO]: [SkinsRestorer] Unsupported plugin (ViaVersion or ProtocolSupport) detected, forcing SpigotSkinRefresher
[13:03:43] [Server thread/INFO]: [SkinsRestorer] Using SpigotSkinRefresher
[13:03:43] [Server thread/INFO]: [SkinsRestorer] Detected Minecraft v1_17_R1, using SpigotSkinRefresher.
```

- [ ] move messages to debug as the refreshers seem to be now both stable.
- [ ] change the working of "Unsupported plugin" as this is not true.
- [ ] "Using SpigotSkinRefresher" is abundant if "Detected Minecraft v1_17_R1, using SpigotSkinRefresher." is thrown

**Comment by Asmodeux18** [comment]
**Hi. I have the same problem, how can i fix that?**

**Comment by xknat** [comment]
fixed by #732
